### DISCUSSION TITLE
Add wget to support healthcheck

### DISCUSSION THREAD

**Discussion by gjrtimmer** [renovatebot/renovate#26089] [Suggest an Idea]
### Tell us more.

I'm using healthchecks to verify that my renovate job has run successfully.
However, `wget` is missing from this image, and because the USER is set to `1000` its not possible to install it.
I could build my own image however, I'm using the gitlab-renovate-runner, and use a mirror repository with automatic updating.

Could you please add `wget` to the image so I can use healthchecks.

**Comment by rarkins** [comment]
The image already contains `curl`
